lib/commands/removeJob-9.lua

```diff
@@ -29,7 +31,9 @@ if not lock then             -- or (lock == ARGV[2])) then
   redis.call("LREM", KEYS[4], 0, ARGV[1])
   redis.call("ZREM", KEYS[5], ARGV[1])
   redis.call("ZREM", KEYS[6], ARGV[1])
-  redis.call("DEL", KEYS[7])
+  redis.call("ZREM", KEYS[7], ARGV[1])
+  redis.call("DECR", KEYS[8])
```

Comment:
Hi @manast. The line `redis.call("ZREM", KEYS[7], ARGV[1])` is just in case people work with auto-assigned IDs and want to re-use a specific ID.
If it would not be a desired behavior, I can remove it from the PR

Reply:
Yes, I mean the line ```redis.call("DECR", KEYS[8])``` which decrements the auto increment job id.